RadarSDK/RadarReplay.m

```diff
@@ -17,6 +17,18 @@ - (instancetype)initWithParams:(NSDictionary *)replayParams {
     return self;
 }
 
+- (instancetype)initWithCoder:(NSCoder *)coder {
```

Comment:
n00b question - should we declare that `RadarReplay` abides by the `NSCoding` interface anywhere, or is that implicit within `NSObject`?